File: MySpace/wwwroot/TestScreenOCR/test/Controller/Controllerfunctions/Delete_Crf_of_User.cs
        public JsonResult Delete_Crf_of_User(string crfId, string remarks)
        {
            try
            {
                int user_crf_delete = 20;
                CRF_Tracker_bll.Delete_Crf_of_User(crfId, remarks, user_crf_delete);
            }
            catch (Exception ex) { throw ex; }

            return Json(1);
        }

File: MySpace/wwwroot/TestScreenOCR/test/js/jsfunctions/Delete_Crf_of_User.js
function Delete_Crf_of_User(crfId) {
    $("#loading").show();
    var remarks = prompt("Please enter remark"); // Show a prompt for remarks

    if (remarks !== null) { // Check if user entered remarks or clicked cancel
        // Perform deletion with remarks
        $.ajax({
            type: "POST",
            url: "/Home/Delete_Crf_of_User",
            data: JSON.stringify({ crfId: crfId, remarks: remarks }),
            contentType: "application/json; charset=utf-8",
            dataType: "json",
            async: false,
            success: function (response) {
                  $("#loading").hide();
                var data = response;
                if (data != null) {
                    alert("Crf Deleted successfully");
                    location.reload();
                }
            }
        });
    }
}